src/views/v2/tickets/item/content/description.php

```diff
@@ -14,7 +14,7 @@
  *
  * @version TBD
  *
- * @var Tribe__Tickets__Ticket_Object $ticket
+ * @var Tribe__Tickets__Ticket_Object $ticket 	Ticket Object
```

Comment:
:no_entry_sign: Spaces must be used for mid-line alignment; tabs are not allowed

<sub>hash: d3dfb452741e64523e2b71d416104f86</sub>

Reply:
```suggestion
 * @var Tribe__Tickets__Ticket_Object $ticket   Ticket Object
```